src/systemjs.config.js

```diff
@@ -30,7 +30,6 @@
     // packages tells the System loader how to load when no filename and/or no extension
     packages: {
       app: {
-        main: './main.js',
```

Comment:
Doesn't work without that:
![image](https://cloud.githubusercontent.com/assets/4172079/22553432/95d6ffea-e954-11e6-9b62-dfea57679d8e.png)